dev/com.ibm.ws.componenttest.2.0/bnd.bnd

```diff
@@ -70,5 +72,6 @@ test.project: true
     com.ibm.websphere.org.osgi.service.component;version=latest,\
     com.ibm.wsspi.org.osgi.service.component.annotations;version=latest,\
     com.ibm.ws.kernel.feature.core;version=latest,\
-    com.ibm.ws.kernel.service;version=latest
+    com.ibm.ws.kernel.service;version=latest,\
+    org.glassfish:javax.json;version=1.1.4
```

Comment:
This should be org.glassfish:jakarta.json;version=2.0.0?

Reply:
https://github.com/OpenLiberty/open-liberty/pull/19478/files#r764190271